exercises/saddle-points/introduction.md

```diff
@@ -1,9 +1,11 @@
 # Introduction
 
-You are planning on building a tree house in the woods near your house so that you can watch the sun rise and set.
+You plan to build a tree house in the woods near your house so that you can watch the sun rise and set.
 
-You've obtained data from a local survey company that shows the heights of all the trees in each rectangular section of the map.
-You need to analyze each grid on the map to find the perfect tree for your tree house.
+You've obtained data from a local survey company that show the height of every tree in each rectangular section of the map.
+You need to analyze each grid on the map to find good trees for your tree house.
 
-The best tree will be the tallest tree compared to all the other trees to the east and west, so that you have the best possible view of the sunrises and sunsets.
-You don't like climbing too much, so the perfect tree will also be the shortest among all the trees to the north and to the south.
+A good tree is both:
+
+- taller than every tree to the east and west, so that you have the best possible view of the sunrises and sunsets.
+- shorter than every tree to the north and south, to minimize the amount of tree climbing.
```

Comment:
The expression "due <direction>" isn't going to be familiar to a lot of people whose primary language isn't English. Maybe "directly" would work?

Reply:
Hm, yes. You're right. Okay, strike that!

Reply:
_Strictly_ north?

> The expression "due " isn't going to be familiar to a lot of people whose primary language isn't English.

I agree that _due_ is likely to be unfamiliar. However, what do you expect will go wrong? I imagine one would either 1) guess correctly or 2) remain mystified, and I deem it unlikely that one would 3) guess _immediately adjacent in the northern direction_ or something.

Anecdotal data:

- I, a non-native speaker, do not remember ever having had trouble with this expression. (But it's likely been ~10 years.)
- My go-to tool [gives me the right answer](https://en.wiktionary.org/wiki/due#English) here, but my simulated dumb googling & Google Translate give awful results.

Reply:
Perhaps a link? Although [this definition](https://www.ldoceonline.com/dictionary/due-north-south-east-west) uses 'directly'.  

But we should stay far away from the `true` (due) vs `magnetic` explanation.  That's confusing to everyone unless you fly a plane or sail a boat or otherwise navigate using a compass.  😄

Reply:
> However, what do you expect will go wrong?

Reading a block of text containing an algorithm (the trees), a mixture of technical terms, and words that may be unfamiliar (e.g. "due") all add up to make someone feel overwhelmed. Our aim is always to provide context that is fun and engaging, but also that doesn't detract or make people feel lost. 

By itself "due" is fine - you can google it etc, but in the context of everything else, each unfamiliar word adds up to more pain and any could be the "straw that breaks the camel's back".

Also - your english is great Matthijs, so even though you're non-native, you're still more comfortable writing than a huge percentage (probably a huge majority) of non-native speakers.

---

In situations like this, I find the word is rarely the problem - it's normally the sentence structure. By moving the context/story to the start and putting the technical details to the end, I suspect we'll get more clarity. For example,

> A good tree:
> 
> - gives the best possible view of sunrises and sunsets, so it should be taller that the trees east and west (the tallest tree on it's row)
> - minimises the amount of tree climbing, so it should be shorter than every tree to the north and south (the shortest tree on its column).

Reply:
@iHiD I like that rewrite.

Reply:
(Btw, Its vs It's - I never know so my brain apparently did one each on those two lines - I highly recommend not copying + pasting that :))

Reply:
🤔 Was _it's_ ever accepted as a (sorta?) genitive? The ball is red. Red is the colour it's.

Turns out [it was](https://en.wiktionary.org/wiki/it%27s#English) 😲

I somewhat expect _it's_ to be accepted again in the future. It makes sense! I wouldn't object (much/long). Way less objectionable than Dutch's [_hun hebben_](https://onzetaal.nl/taalloket/hun-hebben-zij-hebben) (_them have_ vs. _they have_).